scripts/valhalla_build_extract

```diff
@@ -173,11 +223,9 @@ def get_tiles_with_bbox(all_tile_paths: List[Path], bbox_str: str, tiles_dir_: P
         LOGGER.critical(f"Bbox invalid: {list(bbox)}")
         sys.exit(1)
 
-    tile_paths_ = set()
```

Comment:
this used to collect a `set` before. in python, that's an unordered structure, so really we were producing unordered tar files :roll_eyes: my bad.. it's a list now and we deduplicate just before writing it to the tar

Reply:
for reference: we can get duplicate tiles for `--geojson-dir` where multiple polygons intersect the same tile.